src/OpenSage.Game/Graphics/Shaders/NormalMapped.cs

```diff
@@ -57,6 +57,7 @@ public struct MaterialConstantsType
             public float BumpScale;
             public float SpecularExponent;
             public uint /*bool*/ AlphaTestEnable;
+            public uint Padding;
```

Comment:
Please could this be `private readonly float _padding`, same as the padding values in other shaders, and wrapped in a `#pragma` that disables the "unused" warning?

Reply:
Sure will fix